test/azure/AcceptanceTests/zzz_tests.py

```diff
@@ -56,7 +57,7 @@ def test_ensure_coverage(self):
 
         cred = BasicTokenAuthentication({"access_token" :str(uuid4())})
         client = AutoRestReportServiceForAzure(cred, base_url="http://localhost:3000")
-        report = client.get_report()
+        report = client.get_report(platform.python_version())
```

Comment:
so reports are distinguishable since you run with Py27 and Py35 (in the overall report posted on the PR, I take the "worse" report, since I don't think it makes sense to post 2 reports that we really expect to be 100% identical)